doc: added known issue for increased current (#5984)
nrfconnect/sdk-nrf: doc/nrf/known_issues.rst

  **Workaround:** Directed advertising is disabled by default for nRF Desktop.

.. rst-class:: v1-7-1 v1-7-0 v1-6-1 v1-6-0

> NCSDK-12020: Current for Gaming Mouse increased by 1400mA when not in sleep mode.

Reviewer: ```suggestion
NCSDK-12020: Current consumption for Gaming Mouse increased by 1400mA
```

>   Gaming Mouse uses Zephyr driver for PWM_leds which sets pwm every cycle.

Reviewer: ```suggestion
  When not in the sleep mode, the Gaming Mouse reference design has current consumption higher by 1400mA.
  This is because this reference design uses Zephyr's driver for PWM_leds, which sets pwm every cycle.
```
